Jenkinsfile

```diff
@@ -178,6 +178,24 @@ pipeline {
             echo 'Skipping Kafka Streams archetype test for ARM build'
           }
         }
+
+        stage('PowerPC') {
+          agent { label 'ppc64le' }
+          options {
+            timeout(time: 2, unit: 'HOURS')
+            timestamps()
+          }
+          environment {
+            SCALA_VERSION=2.12
```

Comment:
Scala 2.12 support has been deprecated, it would be good to use Scala 2.13 for new stages.

Reply:
Good point, I'll follow up with a few more changes we should do in this area.